pbs/usersync.go

```diff
@@ -32,11 +36,20 @@ const (
 // To get an instance of this from a request, use ParsePBSCookieFromRequest.
 // To write an instance onto a response, use SetCookieOnResponse.
 type PBSCookie struct {
-	uids     map[string]string
+	uids     map[string]temporaryUid
 	optOut   bool
 	birthday *time.Time
 }
 
+// temporaryUid bundles the UID with an Expiration date.
+// After the expiration, the UID is no longer valid.
+type temporaryUid struct {
```

Comment:
The name `temporaryUid` sounds to me like a temporary variable with a short lifetime. Is there a better name for this struct? Possibly `uidWithExpiry`?